### PR TITLE
Vagrant updates

### PR DESCRIPTION
**Pull request by tvlooy**
Everything works but we want to go to 7.1
This is WIP because I'm going to test deb.sury.org first

### CODE REVIEW

**Comment by wimme002** [comment]
We use 7.1 from deb.sury for more than a month now.
Not a single problem encoutered.

**Comment by tvlooy** [comment]
@wimme002 good to know. You know what's going on with dotdeb?

**Comment by wimme002** [comment]
I have no idea. I guess it just isn't released yet.

**Comment by tvlooy** [comment]
I saw this message today and got a bit concerned https://twitter.com/dotdeb/status/818685084752486402

File: shell_provisioner/module/debian.sh [hunk 1]
```diff
@@ -3,26 +3,20 @@
...
-cat << EOF >/etc/apt/sources.list.d/dotdeb.list
-deb http://packages.dotdeb.org wheezy-php56 all
-deb-src http://packages.dotdeb.org wheezy-php56 all
-EOF
-wget -qO - http://www.dotdeb.org/dotdeb.gpg | sudo apt-key add -
-
 # Custom bash prompt
-echo "PS1='[\u@\h-\[\033[00;34m\]dev\[\033[00m\] \w]\n\\$ '" >> /etc/bash.bashrc
-echo "PS1='[\u@\h-\[\033[00;34m\]dev\[\033[00m\] \w]\n\\$ '" >> /home/vagrant/.bashrc
+echo "PS1='[\[\033[00;34m\]\u@readingcorner DEV \[\033[00;31m\]\w$(__git_ps1)\[\033[00m\]]\n\\$ '" >> /etc/bash.bashrc
```
Comment (acrobat) [inline shell_provisioner/module/debian.sh]: Dit moet misschien nog aangepast worden naar secretsanta?

**Comment by tvlooy** [comment]
@acrobat nice catch. Fixed in d33d95283a1369eecdf40cae8a9105611cdddf6b

**Pull request merged**